code/datums/components/scope.dm

```diff
@@ -101,21 +105,36 @@
 	return target_turf
 
 /**
- * We start zooming by hiding the mouse pointer, adding our tracker overlay and starting our processing.
+ * Wrapper for zoom(), so in case we runtime we do not get stuck in a bad state
  *
  * Arguments:
  * * user: The mob we are starting zooming on.
 */
 /datum/component/scope/proc/start_zooming(mob/user)
+	if(zoom(user))
+		zooming = TRUE
+
+/**
+ * We start zooming by hiding the mouse pointer, adding our tracker overlay and starting our processing.
+ *
+ * Arguments:
+ * * user: The mob we are starting zooming on.
+*/
+/datum/component/scope/proc/zoom(mob/living/user)
```

Comment:
why is this on the `living` level? feels like a hurtful typecast when we aren't even sure that it is a living mob or not.